frame/base/bli_winsys.c

```diff
@@ -33,11 +33,11 @@
 */
 
 #ifdef _MSC_VER
-int setenv(const char *name, const char *value, int overwrite) {
+int blis_setenv(const char *name, const char *value, int overwrite) {
     _putenv_s(name, value);
 }
 
-void sleep(int x) {
+void blis_sleep(int x) {
     Sleep(x*1000);
 }
 #endif
```

Comment:
@isuruf Can we change the prefix here to `bli_` instead of `blis_`? Also, where are these functions used? I don't see them in the PR diff. Or maybe you have yet to make those changes?

Reply:
`sleep` is used in the testsuite. Maybe we can move this to a file in testsuite instead of keeping this here.

Reply:
@isuruf I'm fine with keeping them in winsys.c, but regardless of where the definitions reside, would you not also need to update the function call to `sleep()` in the testsuite (so that it calls `blis_sleep()` or `bli_sleep()` instead)?